### DISCUSSION TITLE
TrackingDocument

### DISCUSSION THREAD

**Discussion by Demonchik** [iAlexeyProkhorov/Baroque.NovaPoshta.Client#5] [Q&A] [answered]
Does the project have tracking?

**Comment by iAlexeyProkhorov** [comment]
Hi! Nice question. Looks like no 😂.
I'll add it in next library version). Thank you for your question)

**Comment by iAlexeyProkhorov** [comment, marked as answer]
Hi, Demonchik!
I add tracking method in client 😉.
[Here](https://developers.novaposhta.ua/view/model/a99d2f28-8512-11ec-8ced-005056b2dbe1/method/a9ae7bc9-8512-11ec-8ced-005056b2dbe1) you can find method description.
